utopia-remix/app/models/project.server.ts

```diff
@@ -22,6 +23,26 @@ export async function listProjects(params: { ownerId: string }): Promise<Project
   })
 }
 
+export async function getProjectOwnerById(
+  params: { id: string },
+  config: { includeDeleted: boolean } = { includeDeleted: false },
+): Promise<string | null> {
+  const whereOptions: {
```

Comment:
I changed it to `let`, but I'm wondering how relevant will be a very specific function here in the scope of the file (since if this is the case - we will probably want to support more flags in this generic function)